recipes/fsspec-reference-maker/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "fsspec-reference-maker" %}
+{% set version = "0.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: b40f88718f93508135a8b9e839c0d7880317080a6a26f1128d610e0bc8952f3b
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    # If your project compiles code (such as a C extension) then add the required compilers as separate entries here.
+    # Compilers are named 'c', 'cxx' and 'fortran'.
+    - {{ compiler('c') }}
```

Comment:
Since this is noarch & pure Python, you should remove this section (I assume it came with the template):
```suggestion
```

Reply:
Well I guess this could be fixed in the feedstock... :wink: